src/components/ha-date-input.ts

```diff
@@ -94,6 +94,11 @@ export class HaDateInput extends LitElement {
   }
 
   private _keyDown(ev: KeyboardEvent) {
+    if (["Space", "Enter"].includes(ev.code)) {
+      ev.stopPropagation();
```

Comment:
```suggestion
      ev.preventDefault();
      ev.stopPropagation();
```
I think space would scroll the page without `preventDefault`

Reply:
Yes you're right.